src/commands/ext-install.ts

```diff
@@ -91,6 +91,23 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
         await secretsUtils.ensureSecretManagerApiEnabled(options);
       }
     }
+    const apis = spec.apis || [];
```

Comment:
This probably should happen before we ensureSecretManagerApiEnabled, not after

Reply:
Good catch! Moved ensureSecretManagerApiEnabled after APIs consent.